core/pom.xml

```diff
@@ -140,6 +140,25 @@
 	<build>
 		<pluginManagement>
 			<plugins>
+				<plugin>
```

Comment:
This section needs to be moved out of <pluginManagement> to have an effect...

Reply:
Indeed, thanks for pointing it out.